### PR TITLE
DOC/API: Remove limit and improve docstring.

### PR DESCRIPTION
**Pull request by tacaswell**
smooth over conflicts between EDD and DBA patches.

### CODE REVIEW

**Comment by tacaswell** [comment]
With this PR your branch will merge cleanly into master.

File: dataportal/broker/simple_broker.py [hunk 1]
```diff
@@ -103,20 +103,20 @@ def fetch_events(cls, runs, ca_host=None, channels=None):
...
 
         Parameters
         ----------
-        limit : int, optional
-            The maximum number of headers to find.
-            NOTE: Defaults to 50
-            Maximum number of RunStart Documents to be returned.
         start_time : float, optional
-            timestamp of the earliest time that a run header was created
+            timestamp of the earliest time that a RunStart was created
```
Comment (ericdill) [inline dataportal/broker/simple_broker.py]: To be somewhat thrashy, do we want to call this a "RunStart" since this is the "find_headers" function?

Comment (tacaswell) [inline dataportal/broker/simple_broker.py]: Had not gotten back to this...

Comment (tacaswell) [inline dataportal/broker/simple_broker.py]: and it is probably easiest to clean this up after everything is merged to master to avoid conflicts

**Pull request merged**